### ISSUE TITLE
Set position

### ISSUE THREAD

**Issue by steinjonathan** [ratson/cordova-plugin-admob-free#172]
Is there a way to set the position of the banner?
I want to show it inside a div, on side of some other user infos.

**Comment by MBuchalik** [comment]
I don't think that showing a banner inside a div (or any other HTML element) is possible - the banner isn't rendered in the DOM/WebView. I am not sure if it is currently possible to set a different banner location than very top/bottom of the screen.

**Comment by erperejildo** [comment]
This is not possible. The ad is created on top of the webview

**Comment by kurtiev** [comment]
@steinjonathan Hi, did you find any solution for it?
